apps/publikator/components/editor/modules/meta/PaynotesForm.js

```diff
@@ -50,53 +77,96 @@ const DEFAULT_PAYNOTE = {
   },
 }
 
-const TargetForm = withT(({ t, data, onInputChange }) => (
-  <div>
-    <Interaction.H3>{t('metaData/paynote/form/target/title')}</Interaction.H3>
-    {TARGETS.map((target, i) => {
-      return (
-        <div key={i} style={{ marginBottom: 10 }}>
-          <Label style={{ display: 'block', marginBottom: 5 }}>
-            {t(`metaData/paynote/form/target/${target}`)}
-          </Label>
-          {[true, false, undefined].map((value) => (
-            <Radio
-              key={String(value)}
-              value={String(value)}
-              checked={data[target] === value}
-              onChange={() => onInputChange({ [target]: value })}
-              style={{ marginRight: 20 }}
-            >
-              {t(`metaData/paynote/form/target/value/${value}`)}
-            </Radio>
-          ))}
-        </div>
-      )
-    })}
-  </div>
-))
-
-export default withT(({ t, editor, node }) => {
+const isEmptyPaynote = (paynote) =>
+  !paynote.content &&
+  !paynote.cta &&
+  !paynote.button.label &&
+  !paynote.button.link &&
+  !paynote.secondary.label &&
+  !paynote.secondary.link
+
+const isEmptyPaynotes = (paynotes) =>
+  paynotes?.length === 1 &&
+  isEmptyPaynote(paynotes[0].before) &&
+  isEmptyPaynote(paynotes[0].after) &&
+  Object.keys(paynotes[0].target).length === 0
+
+const TargetForm = withT(
+  ({ t, isFormat, target, inherit, onTargetChange, onInheritChange }) => (
+    <>
+      <Interaction.H3>{t('metaData/paynote/form/target/title')}</Interaction.H3>
+      {isFormat && (
+        <>
+          <br />
+          <Checkbox
+            checked={inherit}
+            onChange={(_, checked) => {
+              console.log({ checked })
+              onInheritChange(checked)
+            }}
+          >
+            Beitragstörer
+          </Checkbox>
+          <br />
+          <br />
+        </>
+      )}
+      {TARGETS.map((tgt, i) => {
+        return (
+          <div key={i} style={{ marginBottom: 10 }}>
+            <Label style={{ display: 'block', marginBottom: 5 }}>
+              {t(`metaData/paynote/form/target/${tgt}`)}
+            </Label>
+            {[true, false, undefined].map((value) => (
+              <Radio
+                key={String(value)}
+                value={String(value)}
+                checked={target[tgt] === value}
+                onChange={() => onTargetChange({ [tgt]: value })}
+                style={{ marginRight: 20 }}
+              >
+                {t(`metaData/paynote/form/target/value/${value}`)}
+              </Radio>
+            ))}
+          </div>
+        )
+      })}
+    </>
+  ),
+)
+
+export default withT(({ t, editor, node, isFormat }) => {
   const paynotes = node.data.get(PAYNOTE_KEY) || []
+  const paynotesCta = node.data.get(PAYNOTE_CTA_KEY)
+  const [mode, setMode] = useState()
```

Comment:
### one more thing

this `useState` and the `useEffect` below should be refactored away and directly inferred from the cta:
```
const mode = paynotesCta || paynotes?.length ? MODE_KEYS.CUSTOM : MODE_KEYS.AUTO
```
(assuming that none is set explicitly in the future – pending comment from earlier)